packages/k8s/k3s/build.yaml

```diff
@@ -10,7 +10,7 @@ env:
   - INSTALL_K3S_SKIP_SELINUX_RPM="true"
   # Remove any possible "+N" from the end of the version. We only bump that to
   # avoid overwritting existing images when we want the package to be rebuilt.
-  - INSTALL_K3S_VERSION=v{{ regexReplaceAll "\\+\\d+$" .Values.version "" }}+k3s1
+  - INSTALL_K3S_VERSION=v{{ regexReplaceAll "\\+\\d+$" .Values.version "" }}+k3s{{.Values.k3s_version}}
```

Comment:
I don't know why we removed this. Looking at `git blame`, we used to have it and that's the reason why `k3s_version` is still around in the collection.yaml below.

Reply:
I think it is because the updater will update the version but not the revision. So it would use a version but report another?

Reply:
but the updater is already doing a lousy job updating only one of them. If we are manually bumping, then we can as well keep `k3s_version` around since every now and then [a `-k3s2` may appear](https://github.com/k3s-io/k3s/releases/tag/v1.30.2%2Bk3s2).

@mauromorales it's not always `k3s1`, see my link above.

Reply:
Yup I'm aware it's not always 1. If we don't plan to use the auto update at all, then it's ok. Maybe I misunderstood the PR

Reply:
I don't see a way to exclude a dir: https://github.com/Luet-lab/extensions/blob/master/extensions/autobump-github/luet-autobump-github

We can ignore the bumping PR maybe?